src/widgets/MathML.js

```diff
@@ -0,0 +1,19 @@
+import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
+
+export default class PlainTextMath extends createjs.Container {
```

Comment:
The class name should be updated to MathML to avoid a name collision with the class provided by PlainTextMath.js

Reply:
this class is not needed we can create text with PlainMathText so I will delete this file.